templates/example_datacite.yml

```diff
@@ -1,62 +0,0 @@
-# Required fields
```

Comment:
Is this file no longer used anywhere? It has at least not been moved but is just deleted.

Reply:
It was only ever used as an example linked from the README in this repository.  I'll update the README to link to the GIN help pages.